app/views/shared/_navbar.html.erb

```diff
@@ -0,0 +1,25 @@
+<nav class="navbar navbar-expand-lg navbar-light">
+  <%= link_to image_tag('guardian_angel_logo.jpeg', alt: 'Guardian Angel Home', class: 'nav-logo-img'), root_path %>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+  <div class="collapse navbar-collapse" id="navbarNav">
+    <ul class="navbar-nav">
+      <li class="nav-item">
+        <%= link_to root_path, class: "nav-link #{'active' if current_page?(root_path)}" do %>
```

Comment:
This flow control is awesome! Super cool to have conditional nav bar items based on the route